config.js

```diff
@@ -118,6 +118,10 @@ module.exports = {
     waterRegime: '0434dc31-a34e-7158-5775-4694af7a60cf'
   },
 
+  chargeInformationUpload: {
```

Comment:
It's fine but I'm not sure you need this really. It's being exported from `config.billing.srocStartDate`

Reply:
Done